docs/sources/tempo/release-notes/v2-1.md

```diff
@@ -0,0 +1,128 @@
+---
+title: V2.1
+weight: 55
+---
+
+# Version 2.1 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.1.
+
+With this release, Tempo enhances TraceQL so you can select traces based on kind, basic arithmetic functions, average/count, and numbers. In addition, Tempo 2.1 introduces vParquet2, a new experimental Parquet version designed to be more compatible with other Parquet implementations.
+
+>**Note**: For a complete list of changes and enhancements, refer to the [Tempo 2.1 changelog](https://github.com/grafana/tempo/releases).
+
+## Features and enhancements
+
+Some of the most important features and enhancements in Tempo 2.1 are highlighted below.
+
+### Enhancements to TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. TraceQL makes it possible to find traces that otherwise would have been difficult or even impossible to identify with Tempo’s 1.x search.
+
+With this release, we’ve expanded the TraceQL language. In Tempo 2.0, TraceQL allowed users to select traces based upon resource attributes, time, duration, and basic aggregates. In Tempo 2.1, users can also select traces based on the following:
+
+* Enable querying by `kind`, an intrinsic attribute  (PR [2217](https://github.com/grafana/tempo/pull/2217))
+    * ``{ kind = server }`, `{ kind = client }`, `{ kind = consumer }``
+    * ``{ kind = server && duration > 1s }``
+* Support arithmetic expressions (PR [2146](https://github.com/grafana/tempo/pull/2146))
+    * ``{ span.bytesProcessed > 10 * 1024 * 1024 }``
+    * ``{ span.bytesProcessed / 1024 / 1024 > 10 }``
+    * ``{ span.jobs / span.bytes > 3 }``
+* Enable min, max, and sum aggregate filters
+    * ``{} | min(duration) > 1s``
+    * ``{} | max(span.bytesProcessed) &lt; 1024 * 1024``
+    * ``{} | sum(duration) > 1h``
+* Support for more numeric types: Compare floating point numbers and integers, allow floating point numbers for durations, and support negative numbers when doing aggregate filters. (PR [2139](https://github.com/grafana/tempo/pull/2139), [2304](https://github.com/grafana/tempo/pull/2304), [2289](https://github.com/grafana/tempo/pull/2289))
+    * ``{ 1 = 1.0 }` `
+    * ``{ duration > 1.5s }`, `{duration > 1s500ms }``
+    * ``{} | count() > -1``
+
+This update also includes major performance improvements to TraceQL query performance.
```

Comment:
Do we want to include an example of what these performance improvements are?

Reply:
we could link to the PR? it has a lot of benchmarks. maybe interesting to the docs crowd?

https://github.com/grafana/tempo/pull/2119

that might be too much tho

Reply:
Linking directly to the PR is a great way of letting people see the details, so that's a perfect suggestion. 
```suggestion
This update also includes major performance improvements to TraceQL query performance. ([PR 2119](https://github.com/grafana/tempo/pull/2119))
```